### DISCUSSION TITLE
How can I limit concurrent executions on a global level

### DISCUSSION THREAD

**Discussion by boredland** [quirrel-dev/quirrel#812] [Q&A] [answered]
Hi there! When creating queues without a schedule (e.g. to be executed in a FIFO order), I currently only see the option to make all jobs in that queue run sequentially using "exclusive: true" as a default job parameter.
For me, that sounds a bit like an overkill, because my infrastructure is limited, but could easy handle a lot of requests. What I would like to have: a limit on the Quirrel server side about how many concurrent executions to my API endpoints it is going to make. Is there such an option?

**Comment by Skn0tt** [comment, marked as answer]
Hi @boredland! `exclusive` is indeed not meant for limiting load on your application, but for cases where race conditions need to be prevented. Quirrel doesn't have an option for limiting concurrent execution, as it's built with serverless deployments in mind, where infrastructure scale isn't a concern.
